pages/linux/lastb.md

```diff
@@ -0,0 +1,23 @@
+# lastb
+
+> Show a listing of last logged in users.
+
+- Show a list of all last logged in users:
+
+`lastb`
+
+- Show a list of all last logged in users since:
+
+`lastb --since {{time}}`
```

Comment:
It becomes much clear if you mention the time format. I struggled with it and had to look up the man page. You can mention - "YYYY-MM-DD" or "hh:mm" also if you prefer.